JavaScript/JavaScriptSDK/Context/User.ts

```diff
@@ -11,6 +11,11 @@ module Microsoft.ApplicationInsights.Context {
         public id: string;
 
         /**
+         * Authorized user id
+         */
+        public authId: string;
```

Comment:
But since with the `authId` we are expecting the server side to respect whatever `string` the user might set to get a more accurate count, i think that this should not be exposed publicly.